contentcuration/contentcuration/templates/base.html

```diff
@@ -29,6 +29,19 @@
 			<script type="text/javascript" src="{% url 'javascript-catalog' %}"></script>
 			{% endblock head %}
 
+      {% block analytics %}
+        {% if not debug %}
```

Comment:
Does this need to be passed in through the views files, or is this automatically added to the page by Django?

Reply:
Sweet